ast-nodes/parameter.dart

```diff
@@ -47,7 +47,6 @@ class Parameter implements Node {
   }
 
   Pointer<LLVMOpaqueValue> generateCode(Module module) {
-    // TODO: implement
-    return null;
+    return scopeMark.resolve(name).opaqueValue;
```

Comment:
This will not find the parameter, and don't bother, just leave this method to return `null`, we won't ever call it